bin/ofs/Cargo.toml

```diff
@@ -50,6 +47,16 @@ tokio = { version = "1.37.0", features = [
   "signal",
 ] }
 url = "2.5.0"
+tap = "1.0.1"
+
+[target.'cfg(any(target_os = "linux", target_os = "freebsd"))'.dependencies]
+fuse3 = { "version" = "0.7.2", "features" = ["tokio-runtime", "unprivileged"] }
+libc = "0.2.154"
+nix = { version = "0.28.0", features = ["user"] }
+
+[target.'cfg(target_os = "windows")'.dependencies]
+wincs = { git = "https://github.com/ok-nick/wincs" }
```

Comment:
Hi, please don't depend on a git version which will prevent us from releasing and handling licenses.

Reply:
You are right, I will draft this PR until it releases.